### DISCUSSION TITLE
Donald duck quick going problem 3d

### DISCUSSION THREAD

**Discussion by win32forever** [joncampbell123/dosbox-x#5168] [General]
hello everyone good morning, I have a problem with dosbox x with donald duck when I start it it gives me an error setup directx6 driver dislay direct3d hal how can I solve it? I also installed the voodoo video card drivers

**Comment by BridgeHeadland** [comment]
According to this page -> https://www.pcgamingwiki.com/wiki/Donald_Duck:_Goin%27_Quackers the game requires any 3D accelerators, something DOSBox-X cannot emulate as of today, currently it only emulates 2D accelerators, like S3 ViRGEV/X and ATI Mach64, although ATI's 2D accelerators are completely experimental, and don't quite work as they should, at least not in old-school Windows.
True, some games with 3D graphics can be run with 2D accelerators, to some extent.
There is already an issue for that -> https://github.com/joncampbell123/dosbox-x/issues/5040 :)

**Comment by win32forever** [comment]
``now everything is clear and I thank you very much I hope in the future a 3D acceleration is inserted such as a 16 mb or 64 graphics card thanks for your work
